content/docs/conditional-rendering.md

```diff
@@ -24,7 +24,7 @@ function GuestGreeting(props) {
 }
 ```
 
-We'll create a `Greeting` component that displays either of these components depending on whether a user is logged in:
+हम `Greeting` कॉम्पोनेन्ट बना रहे हे । अगर यूजर लॉगिन हे तभी देखेगा :
```

Comment:
`बना रहे हे । अगर यूजर लॉगिन हे तभी देखेगा :` => `बना रहे है। अगर यूजर लॉगिन हे तभी देखेगा:`

Reply:
not fixed

Reply:
`कॉम्पोनेन्ट` => `कौम्पोनॅन्ट`

Reply:
`देखेगा :` => `देखेगा:`

Reply:
> `देखेगा :` => `देखेगा:`

this is missed

Reply:
> `कॉम्पोनेन्ट` => `कौम्पोनॅन्ट`

this is missed

Reply:
> `देखेगा :` => `देखेगा:`

this is missed again